gdk/gdk.go

```diff
@@ -1801,10 +1774,8 @@ func PixbufAnimationNewFromFile(filename string) (*PixbufAnimation, error) {
 		return nil, errors.New(C.GoString((*C.char)(err.message)))
 	}
 
-	obj := &glib.Object{glib.ToGObject(unsafe.Pointer(c))}
-	p := &PixbufAnimation{obj}
-	runtime.SetFinalizer(p, func(_ interface{}) { obj.Unref() })
-	return p, nil
+	obj := glib.Take(unsafe.Pointer(c))
```

Comment:
[`gdk_pixbuf_animation_new_from_file`](https://developer.gnome.org/gdk-pixbuf/stable/gdk-pixbuf-Animations.html#gdk-pixbuf-animation-new-from-file) returns "A newly-created animation with a reference count of 1, or NULL", which means we do not call `Ref()`, just ensure that `Unref()` is called.

Reply:
done